lib/posthog/client.rb

```diff
@@ -87,6 +98,36 @@ def queued_messages
       @queue.length
     end
 
+    def is_feature_enabled(flag_key, distinct_id, default_value=false)
+      if !@personal_api_key
```

Comment:
`unless`, wow, quite the vocab ruby has

Reply:
yeah, ruby is a "write anything and it usually works the first time" type of language. Rails adds even more stuff with ActiveSupport, such as `[1,2,3].third == 3`...

I personally dislike this verbosity, as it makes it hard to read code, as you never know which of the 10 ways this can be done is selected... :)